tests/test_wand.py

```diff
@@ -228,6 +230,30 @@ def test_get_wand_image(self):
 
         self.assertIsInstance(wand_image, _wand_image().Image)
 
+    @unittest.skipIf(no_avif_support, "imagemagic was not built with avif support")
```

Comment:
```suggestion
    @unittest.skipIf(no_avif_support, "ImageMagick was built without AVIF support")
```

Reply:
I know you copied previous comments, but it is important to try to be correct. If you have the time and can, it would be great to update the other comments in this test file as well